### PR TITLE
Fix Admin client cannot connect when TLS is enabled

### PR DESCRIPTION
**Pull request by hasalex**
Fixes issue #75

### CODE REVIEW

File: src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java [hunk 1]
```diff
@@ -20,17 +20,27 @@
 import org.jboss.shrinkwrap.api.exporter.ZipExporter;
 import org.jboss.shrinkwrap.api.importer.ExplodedImporter;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jetbrains.annotations.Nullable;
```
Comment (dasniko) [inline src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java:23]: I know that these checks might be good and important. But for now I haven't used them yet in this project and I rather would introduce them more holistic than with single PRs. Please remove this annotation. If you feel this project should use them more widely, open an issue for this. Thanks.

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java [hunk 2]
```diff
@@ -4,11 +4,15 @@
...
 import org.junit.jupiter.api.Test;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.resource.ServerInfoResource;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.startsWith;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```
Comment (dasniko) [inline src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java:15]: This import is unused.

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java [hunk 3]
```diff
@@ -69,6 +73,17 @@ public void shouldThrowNullPointerExceptionUponNullTlsKeystore() {
...
+    @Test
+    public void shouldAdminClientBeAbleToConnect() {
+        try (KeycloakContainer keycloak = new KeycloakContainer().useTls()) {
+            keycloak.start();
+
+            Keycloak admin = keycloak.getKeycloakAdminClient();
+            ServerInfoResource serverInfoResource = admin.serverInfo();
+            assertNotNull(serverInfoResource.getInfo());
+        }
+    }
```
Comment (dasniko) [inline src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java:85]: Good first test case. Please add another two tests, one for `.useTls(tlsCertificateFilename, tlsCertificateKeyFilename)` and one for `.useTlsKeystore(tlsKeystoreFilename, tlsKeystorePassword)` like the already existing TLS basic tests are available.
In a quick manual test, at least the `.useTls(String, String)` test failed, so there might be additional work to be done.

**Review by dasniko — CHANGES_REQUESTED** [review]
Great addition/fix, thank you for your work, it is really appreciated! 🙏

However, I added some comments to your code. It would be good to fix them. 🙂

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java (hunk 3 above)
Comment (hasalex) [reply src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java:85]: Indeed, the certificate case was not in my personal need.
I'll fix it too.

**Review by dasniko — APPROVED** [review]

**Comment by dasniko** [comment]
Thanks @hasalex 🙏

**Pull request merged**